+mexopencv/make.m

```diff
@@ -436,3 +434,16 @@ function check_path_mexopencv(opts)
         end
     end
 end
+
+%%
+%% Return: true if the environment is Octave.
+%%
+function retval = isOctave
+    persistent cacheval;  % speeds up repeated calls
+
+    if isempty (cacheval)
+        cacheval = (exist ("OCTAVE_VERSION", "builtin") > 0);
```

Comment:
you cant have double-quoted strings in MATLAB (only Octave), you gotta make them single-quotes...